plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -252,6 +253,8 @@
     protected String _rngPath = "/dev/random";
     protected int _rngRatePeriod = 1000;
     protected int _rngRateBytes = 2048;
+    protected File _qemuSocketsPath;
+    private String _qemuGuestAgentSocketName = "org.qemu.guest_agent.0";
```

Comment:
`_qemuGuestAgentSocketName` appears to be a constant value.  If so, please consider declaring it as a 'private static final`.